.github/workflows/check-deps.yml

```diff
@@ -0,0 +1,18 @@
+name: Check for dependency changes
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '30 */12 * * *'
```

Comment:
Since this is a public repo (we're not limited with GA minutes), you can set it to run hourly.